app/views/claims/service_updates/content/claim-window-closed.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Claim window closed on 19 July
+date: 2024-07-20
+start_page: true
+---
+
+# Claim window closed on 19 July
+
+20 July 2024
+
+Mud minnow featherfin knifefish brook lamprey Ganges shark gombessa Kafue pike. Butterflyfish orangestriped triggerfish European eel straptail moray eel green swordtail pompano half-gill, sand tilefish roosterfish.
```

Comment:
Is this... real content? 🤔 _Should this_ be real content? 😅

Reply:
No. This shouldn't be merged yet. Pending real content.

![CleanShot 2024-07-30 at 13 16 25](https://github.com/user-attachments/assets/35cab54e-d36a-4bb3-b9c6-1a2b55062aed)

😉